### ISSUE TITLE
Не работает больше расширение

### ISSUE THREAD

**Issue by jahlib** [JustKappaMan/VK-Video-Downloader#17]
Вк сменил домен на https://vkvideo.ru/ я докинул домен в манифест но не заработало

**Comment by JustKappaMan** [comment]
@jahlib последнюю версию используешь? Я обновлял пару недель назад как раз под vkvideo.ru. Если и она не работает - позже гляну.

**Comment by jahlib** [comment]
Да, вот только что перекачал и перепроверил, подумал у меня устарела

**Comment by Korb** [comment]
I checked, for videos in the domain vkvideo.ru the script works properly. Average download speed is 1.15 MB/s.

VK-Video-Downloader-desktop 1.1.11
Tampermonkey 5.3.3 (December 20, 2024)
Mozilla Firefox 135.0b2 (64-bit)
Microsoft Windows 10.0.19045.5247

**Comment by VerMishelb** [comment]
Для vkvideo.ru домен изменён, а вот для vk.ru — нет, поэтому видео с обычного ВК не дают ссылок на скачивание. Оба домена работают, переадресации нет. Почему у меня в закладках vk.ru оказался, для меня пока загадка.

**Comment by JustKappaMan** [comment]
@VerMishelb добавил поддержку [vk.ru](https://vk.ru), спасибо за напоминание о существовании этого домена :slightly_smiling_face:

**Comment by JustKappaMan** [comment]
@jahlib проверил, расширение работает на [vk.com](https://vk.com), [vk.ru](https://vk.ru) и [vkvideo.ru](https://vkvideo.ru).

* Linux Mint 21.3 x86_64
  * Google Chrome 131.0.6778.264 (Official Build) (64-bit)
  * Firefox 134.0 (64-bit) & Tampermonkey 5.3.3
    * VK-Video-Downloader-desktop 1.1.12

Возможно открываете видео в каком-то не совсем типичном разделе сайта. Если проблема сохраняется и есть более подробная информация о том, как её воспроизвести - откройте новую issue, предоставив эту информацию.

Эту я закрываю.
